changelog.d/11381.doc

```diff
@@ -0,0 +1 @@
+Quote wildcard `federation_certificate_verification_whitelist` in config
```

Comment:
CI is complaining because this doesn't end with a fullstop. Also, we prefer to write these notes in the "imperative"a way that describe the user-visible change. We're also happy to credit you in the commit message to.

```suggestion
Fix incorrect example configuration for `federation_certificate_verification_whitelist`. Contributed by @NicolaiSoeborg.
```

Reply:
I think the changelog should be better now :)